### PR TITLE
fix CI check for apispec changes

### PR DESCRIPTION
**Pull request by engelmi**
This PR enables the check for any API changes and the re-generation of relevant python binding files (e.g. `setup.py` with the proper version).

### CODE REVIEW

File: build-scripts/build-bindings.sh [hunk 1]
```diff
@@ -4,9 +4,7 @@
 # Important note: 
 # Run from root directory
 
-# Parse package version from the project
-meson setup builddir
-VERSION="$(meson introspect --projectinfo builddir | jq -r '.version')"
+VERSION_SCRIPT=$(dirname "$(readlink -f "$0")")/version.sh
```
Comment (mwperina) [inline build-scripts/build-bindings.sh]: Is there a reason to replace `VERSION` with `VERSION_SCRIPT`? It seems to me less clear ...

Comment (engelmi) [reply build-scripts/build-bindings.sh]: No real reason. I changed it to directly retrieve the version. 
PTAL @mwperina

**Review by mwperina — APPROVED** [review]
LGTM

**Pull request merged**